src/components/v2/V2Project/NewDeployModal.tsx

```diff
@@ -61,6 +66,25 @@ export default function NewDeployModal({
       <div>
         <RichButton
           prefix="1"
+          heading={<Trans>Set a project handle (optional)</Trans>}
```

Comment:
Adds "set handle" step to newDeploy modal, which opens drawer with form to configure project handle